draft-westerlund-tsvwg-sctp-crypto-chunk.md

```diff
@@ -738,6 +744,10 @@ containing the Protection Engine previously sent as protected
 association parameter in INIT-ACK chunk, it will clear the T-valid
 timer and will move into ESTABLISHED state.
 
+Once in ESTABLISHED state, only CRYPTO chunks can be sent to the remote peer
+and any other type of plain text SCTP chunks coming from the remote
```

Comment:
Just checking that it is fine to throw away all plain text CHUNKS. Including ASCONF?

Reply:
Also section 2.1 mentions SHUTDOWN-COMPLETE chunk. Is that wrong or do that need to be mentioned here?